server/server.go

```diff
@@ -253,6 +255,12 @@ func (server *Server) PlayerProvider(provider player.Provider) {
 	server.playerProvider = provider
 }
 
+// LoadResourcePack loads a resource pack to the server. The pack will eventually be sent to clients who join the
+// server when started.
+func (server *Server) LoadResourcePack(pack *resource.Pack) {
```

Comment:
This doesn't actually load them. I'd call this AddResourcePack.

Reply:
Changed.